bin/run.sh

```diff
@@ -167,10 +158,6 @@ case ${option} in
 	supportNotificationTest	| tee $SUPPORT_NOTIFICATION_LOG_PATH
     ;;
   	-exc)
-    echo "Info: Initiating ExportClient Test"
-    exportClientTest | tee $EXPORTCLIENTLOGSPATH
-    ;;
-    -ss)
```

Comment:
`-ss)` should be kept, but remove `-exc)`

Reply:
Great catch!